AmericaTVGoAuthProvider/AmericaTVGoAuthProvider.swift

```diff
@@ -8,14 +8,13 @@
 
 import ApplicasterSDK
 import ZappPlugins
-import ZappLoginPluginsSDK
 
 public typealias AmericaTVGoAuthProviderCompletion = ((ZPLoginOperationStatus) -> Void)
 
 let AmericaTVGoRegisterLaterNotification = Notification.Name("AmericaTVGoRegisterLaterNotification")
 let AmericaTVGoCancelAuthenticationNotification = Notification.Name("AmericaTVGoCancelAuthenticationNotification")
 
-class AmericaTVGoAuthProvider: NSObject, APAuthorizationClient, ZPAppLoadingHookProtocol, ZPBaseLoginProviderFlowHandler {
```

Comment:
Shouldn't we replace ZPBaseLoginProviderFlowHandler with some new interface? (Waiting answer from the DRT)